src/storage/txn/store.rs

```diff
@@ -170,6 +170,18 @@ impl TxnEntry {
             _ => unreachable!(),
         }
     }
+    /// This method will generate this kv pair's key
+    pub fn form_key(&self) -> Result<Vec<u8>> {
+        match self {
+            TxnEntry::Commit {
+                default: _, write, ..
+            } => Ok(Key::from_encoded(write.0.clone())
+                .truncate_ts()?
+                .into_raw()?),
+            // Prewrite are not support
+            _ => unreachable!(),
+        }
+    }
```

Comment:
```suggestion
    pub fn as_key(&self) -> Result<&[u8]> {
        match self {
            TxnEntry::Commit { write, .. } => Ok(Key::truncate_ts_for(&write.0)?),
            // Prewrite are not support
            _ => unreachable!(),
        }
    }
```

not sure if we should ignore the `default` cf

Reply:
From @3pointer, the keys of `default` and `write` are the same.